R/get.samples.R

```diff
@@ -28,11 +40,10 @@
 #'  \item neighbouring year
 #' }
 ##' @import dplyr lubridate tidyr
-##' @importFrom nnet multinom
 ##' @importFrom data.table rbindlist
```

Comment:
I found out bind_rows() do the same thing. Using it would eliminate the need for `data.table::rbindlist`

Reply:
We already use the data.table package for a lot of other things, and the data.table function as well (in read.nafoB I think).  Given that data.table is usually faster than tidyr and I think more stable, it's more logical to keep rbindlist?